modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/Diagram.js

```diff
@@ -69,12 +66,13 @@ const Diagram = ({
 			cy: 0,
 			id: null,
 			label: '',
-			linked_to_sku: 'sku',
+			linkedToSku: 'sku',
```

Comment:
We would need to add this as a constant somewhere. Either at the top of this file or in a separate constants file


```suggestion
			linkedToSku: SKU,
```